src/Microsoft.AspNetCore.SystemWebAdapters/Adapters/RemoteAppOptions.Shared.cs

```diff
@@ -37,5 +38,11 @@ public class RemoteAppOptions
     /// </summary>
     [Required]
     public Uri RemoteAppUrl { get; set; } = null!;
+
+    /// <summary>
+    /// Gets or sets an HttpClient to use for making requests to the remote app.
+    /// A new HttpClient will be automatically generated if none is provided.
+    /// </summary>
+    public HttpClient? BackchannelHttpClient { get; set; }
```

Comment:
Before, the timeout was on auth only. Is this something that can be shared now for all remote app stuff?

Reply:
Yes, I think this makes sense for all of them since it translates into an HttpClient timeout which is common to all of those scenarios.